src/SimpleVideoCutter/MainForm.cs

```diff
@@ -615,21 +596,121 @@ private void StoreSettings()
             VideoCutterSettings.Instance.StoreSettings();
         }
 
-        private void EnqeueNewTask()
+
+        private void UnloadVideo()
+        {
+            if (vlcControl1.MediaPlayer != null)
+            {
+                vlcControl1.MediaPlayer.Stop();
+                vlcControl1.MediaPlayer.Media?.Dispose();
+                vlcControl1.MediaPlayer.Media = null;
+            }
+
+            if (videoViewHover.MediaPlayer != null)
+            {
+                videoViewHover.MediaPlayer.Stop();
+                videoViewHover.MediaPlayer.Media?.Dispose();
+                videoViewHover.MediaPlayer.Media = null;
+            }
+        }
```

Comment:
Probably, UnloadVideo() can be removed completely, as OpenNextFileInDirectory() does all things necessary.  I added the explicit unloading, because I once had a crash, and I noticed that OpenNextFileInDirectory() only implicitly unloaded the video. I didn't like that. Btw, setting fileBeingPlayed to null before calling OpenNextFileInDirectory() would require adjustments to that method. Let's save us the hassle.

Reply:
I would love to keep the `UnloadVideo`, as it would also fix the issue #18. I can imagine that closing a file (to make sure some other program can have exclusive access) may be useful in some workflows. 
I think the changes I suggested would be enough (haven't checked though). 

> btw, setting fileBeingPlayed to null before calling OpenNextFileInDirectory() would require adjustments to that method. Let's save us the hassle.

Yes, currently this would be an issue, but I think adding a `lastFileThatWasPlayed` variable would solve this issue nicely and should be rather straightforward,

Reply:
`UnloadVideo()` is ready to be used now.